aws-transfer-user/contract-tests-artifacts/dependencies.yml

```diff
@@ -0,0 +1,56 @@
+---
+Resources:
+  UluruAccessForTransfer:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: transfer.amazonaws.com
+            Action: sts:AssumeRole
+
+  UluruTransferEfsServer:
+    Type: 'AWS::Transfer::Server'
+    Properties:
+      Domain: EFS
+      EndpointType: PUBLIC
+      IdentityProviderType: SERVICE_MANAGED
+      Protocols:
+        - SFTP
+
+  UluruTransferS3Server:
+    Type: 'AWS::Transfer::Server'
+    Properties:
+      Domain: S3
+      EndpointType: PUBLIC
+      IdentityProviderType: SERVICE_MANAGED
+      Protocols:
+        - SFTP
+
+#  TestUser:
```

Comment:
Do we want to keep this in?

Reply:
Oh, that was some unrelated test code. I should delete it.